docx/text/paragraph.py

```diff
@@ -39,6 +41,77 @@ def add_run(self, text=None, style=None):
             run.style = style
         return run
 
+    def add_field(self, instrText=None):
+        """
+        Adds new field `w:fldChar` to run. Pass `instrText` to specify
+        filed instruction.
+        """
+        self.add_run().add_fldChar()
+        if instrText:
+            self.add_run().add_instrText(instrText)
+        self.add_run().add_fldChar(fldCharType='end')
```

Comment:
Maybe this should not be part of this pull request

Reply:
yes, same as previous, good catch